views/includes/save-and-continue-button.html

```diff
@@ -1,6 +1,6 @@
 {% set textContinueButton = "Continue" %}
 
-{% if OE_CONFIGS.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 == "true" or OE_CONFIGS.FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME == "true" %}
+{% if (isSaveAndContinue === undefined and OE_CONFIGS.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 === "true") or isSaveAndContinue === true %}
```

Comment:
I don't understand the logic here. I see no mention of the new feature flag for Save&Continue in Update.